src/formatter/codeframeFormatter.ts

```diff
@@ -22,6 +22,16 @@ export function createCodeframeFormatter(options: any) {
       : colors.bold.red;
     const positionColor = colors.dim;
 
+    if (message.isInternalType()) {
```

Comment:
We won't need this anymore

Reply:
Unfortunately I think we still do - internal errors have no useful file, line or column to display, but they have a stack trace. So those still need to be displayed separately.

I could try to parse the file name, line and column from the stack trace, but given that's just a string that might fail in different node versions if the formatting changes or something alike. Also, a stack trace has multiple filenames/lines/columns, so without separate handling that extra information might get lost.

Reply:
These are valid points....

Hmmmm......

I really want to avoid having another error type because of consumers like `create-react-app` etc rolling their own formatters. 

Given that, as I understand it, this the edgiest of edge cases.... i.e we don't expect this to happen once in a blue moon then perhaps it's not the end of the world.

What's your view?

Reply:
I think it doesn't hurt this way. The `stack` is additional information and `file`, `line` and `character` are optional properties, so every formatter should have to check for those to exist anyways (and in the worst case, it would display `error in file [internal], undefined:undefined`).

With a formatter not supporting this, the users would see the error and that it occured, they might just not get all the information available.

The only thing I'm not 100% sure about is that `file` would currently contain `[internal]`, which might pose problems if the formatter just tries to read the file without checking for it's existence.

Reply:
> The only thing I'm not 100% sure about is that file would currently contain [internal], which might pose problems if the formatter just tries to read the file without checking for it's existence.

I'm pretty sure it doesn't. Let's go with this.